### ISSUE TITLE
Shoes wikipedia article is outdated

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#297]
Take a look at [Shoes (GUI toolkit)](https://en.wikipedia.org/wiki/Shoes_(GUI_toolkit)). The article is overly out of date. It would be a good thing to add a **latest release** with version 3.3 information (and possibly upcoming 3.4. A feature list, including new SVG, would be a good list.

**Comment by ccoupe** [comment]
It's not that out of date, except for 3.2 and 3.3 and the change of 3.x github address. Looks like someone on the Shoes 4 team wrote it - I could guess who. I don't have an account at wikipedia, if someone else does, go ahead and fix it.  No plans for 3.4 - it would take some thing spectacular and a lot more contributers - maybe a working OSX port of GTK3 and some way to meld GApplication clase into Shoes. Plenty of version numbers with 3.3.n -yes I know thats not what the 3 digit is for but we are capped below 4.0

**Comment by BackOrder** [comment]
Oops. I should say the upcoming **3.3.2**.  No account needed to edit Wikipedia. The point is Shoes is alive but it's not reflected in the article. Features are also not very clear; the current article seems like Shoes is like Logo programming language. Draw lines, draw circles, etc.

**Comment by ccoupe** [comment]
Agree, it's not what I would have written - it's not like there is a size limitation.

**Comment by ccoupe** [comment]
I got tired of chasing missing `end` and got a wikipedia account and fixed somethings, my way. For better or worse.

**Comment by BackOrder** [comment]
Already better. Thanks.
